packages/reputation-miner/ReputationMiner.js

```diff
@@ -40,7 +40,7 @@ class ReputationMiner {
         network_id: 515,
         vmErrorsOnRPCResponse: false,
         locked: false,
-        logger: console,
+        logger: { log: x => console.log("Ganache:", x)},
```

Comment:
What's this doing?

Reply:
Making it obvious that the output being logged is coming from Ganache, not from a hardhat node somewhere, which I lost time to when running down what was happening.